eve/main.yml

```diff
@@ -7,5 +7,27 @@ branches:
 stages:
   pre-merge:
     worker:
-      type: local
-    steps: []
+      type: kube_pod
+      path: eve/workers/pod-builder/pod.yaml
+      images:
+          docker-builder: eve/workers/pod-builder
+    steps:
+    - Git: &git_pull
+        name: git pull
+        repourl: "%(prop:git_reference)s"
+        mode: full
```

Comment:
Not necessarily.

Reply:
which part is not ?

Reply:
A full clone instead of a shallow one. One more ping @tcarmet

Reply:
Yep, not necessary indeed.